### ISSUE TITLE
Equipment descriptions missing for all weapons and for Rogue off-hand gear (shields)

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#9987]
As reported by Kent the Apprentice (b82b7aa8-161f-4b02-aef9-4a319e47f7b2) in the Report a Bug guild:

"The equipment description is missing for all rogue offhand item (class specific items, non seasonal) and for all mainhand items in my android phone! (tried reloading content) Thanks :)

_edit: This is an API bug not an Android bug_

**Comment by Alys** [comment]
If this turns out to be an API problem, tell me and I'll log this in the main repo instead.

**Comment by TheHollidayInn** [comment]
Did you mean to log this here?

**Comment by Alys** [comment]
Ah geeeeez, no, sorry, thanks! :)

**Comment by Alys** [comment]
Reopening. It's an API problem. As @phillipthelen writes in the Android issue:
"This is not a client issue. Looking at the content API, all of the default class weapons and shields are missing their descriptions (possibly also other gear, but that's the one where I noticed it)"

I've adjusted the [equipment table creator.html](http://oldgods.net/habitica/equipment_table_creator.html) which reads from the content api to show the problem (scroll down to below the second white textarea):
```
WARNING:
No notes for weapon_warrior_0
... etc ...
No notes for weapon_rogue_0
... etc ...
No notes for weapon_healer_0
.... etc ...
No notes for weapon_wizard_0
... etc ...
No notes for weapon_special_0
No notes for weapon_special_1
No notes for weapon_special_2
No notes for weapon_special_critical
... etc ...
No notes for weapon_mystery_201411
... etc ...
No notes for weapon_armoire_basicCrossbow
... etc ...
No notes for weapon_armoire_scepterOfDiamonds
No notes for shield_rogue_0
No notes for shield_rogue_1
No notes for shield_rogue_2
No notes for shield_rogue_3
No notes for shield_rogue_4
No notes for shield_rogue_5
No notes for shield_rogue_6
```
It's only a problem for weapons, not other gear. 
The Rogue shields reuse the Rogue weapon notes, so that's why they're failing.

I'm guessing it's related to this code: https://github.com/HabitRPG/habitica/blob/develop/website/common/script/content/gear/weapon.js

**Comment by TheHollidayInn** [comment]
Ah, that's my bad then. I'll fix it.

**Comment by paglias** [comment]
Fixed in #10005
